<?xml version="1.0"?>
<launch>

  <env name="ROSCONSOLE_CONFIG_FILE"
       value="$(find art_pr2_grasping)/config/rosconsole.conf"/>

  <arg name="debug" default="false" />
  <arg unless="$(arg debug)" name="launch_prefix" value="" />
  <arg     if="$(arg debug)" name="launch_prefix" value="gdb --ex run --args" />

  <!-- Launch pick and place -->
  <node name="pr2_grasping" launch-prefix="$(arg launch_prefix)" pkg="art_pr2_grasping" 
	type="pr2grasp" output="screen" respawn="false">
    <rosparam command="load" file="$(find art_pr2_moveit_config)/config/kinematics.yaml"/>
    <rosparam command="load" file="$(find art_pr2_grasping)/config/pr2_grasp_data.gazebo.yaml"/>
    <rosparam param="groups">["left_arm", "right_arm"]</rosparam>
    <rosparam param="gripper_state_topics">["/l_gripper_controller/state", "/r_gripper_controller/state"]</rosparam>
    <rosparam param="default_poses">["tuck_left_arm", "tuck_right_arm"]</rosparam>

  </node>

</launch>
